website/src/components/Search/Modal.tsx

```diff
@@ -40,7 +58,15 @@ export const SearchModal = () => {
         aria-label="Search fonts"
         role="search"
       >
-        Search
+        <Box>
+          <SearchIcon ml="-5px" mr="10px" />
+          Search
+        </Box>
+
+        <Box>
```

Comment:
I think it'd be good to hide this when the website switches to mobile view. 

Also, it does feel vertically unaligned. Am I the only one seeing that?